rbi/core/basic_object.rbi

```diff
@@ -1,9 +1,72 @@
 # typed: __STDLIB_INTERNAL
 
+# [BasicObject](BasicObject) is the parent class of
```

Comment:
Those links are imperfect, I know. 
I ran out of my 2-hour timebox of making this better.
Unless there's blocking feedback, I'd prefer to merge as-is  and get to other work :-)